### PR TITLE
USAA does not support hardware tokens or email

### PR DESCRIPTION
**Pull request by pconwell**
USAA only supports three 'levels' of security: normal username and password, username and password with an SMS OTP, or username with PIN+OTP generated by a symantec app.

### CODE REVIEW

**Comment by Carlgo11** [comment]
@pconwell according to #822 and #3345 they do support hardware and email 2FA. Are you saying that that's changed?

**Comment by pconwell** [comment]
In #822, they are referring to Symantec VIP tokens which are *not* hardware tokens. They are software tokens identical to standard OTP tokens that happen to run on Symantec "hardware". The "hardware tokens" they are identical to a smart phone OTP app, except instead of running on a smart phone they run on a Symantec VIP token. Additionally, Symantec no longer offers the actual physical tokens and the tokens were replaced by a smart phone app anyway - so I would not consider it a hardware token. At one point in time, Symantec *did* have physical software tokens, but 1) even back then it was still a software token by all intents and purposes, and 2) since they have switched to smart phone apps, it is a software token in every sense  of the word today.

As far the email OTP, their website does not mention email and even calls it "CyberCode Text" - however @coleslaugh suggests that they now include email in the 'text' codes. I haven't tested it, but I can if needed.

In summary - USAA has never supported hardware tokens, and USAA *might* support email OTP codes.

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
Alright, thanks for the clarification @pconwell. 
Could you remove the lines with `No` as the value? Values are false by default and so it's not necessary to specify (plus it also breaks the build).

**Comment by pconwell** [comment]
I've edited the pull request (I think... I'm not an expert on git). I also confirmed that USAA *does* support email 2FA as long as the user explicitly enables email as a backup recovery method, so I made that change as well. So the only actual change is removing the hardware token from USAA.

**Review by Carlgo11 — APPROVED** [review]

**Review by RichJeanes — APPROVED** [review]
